### PR TITLE
Add RepositoryPolicies doc

### LINKED ISSUE #1658: Document the available options for Maven "update-policy"
As a user, I'd like to know all the `update-policy` options applicable for maven artifacts. Perhaps we could point to this link [here](https://maven.apache.org/ref/3.5.0/maven-artifact/apidocs/org/apache/maven/artifact/repository/ArtifactRepositoryPolicy.html)?

The current [docs](https://docs.spring.io/spring-cloud-dataflow/docs/1.3.0.M2/reference/htmlsingle/#getting-started-maven-configuration) on this subject only point to the steps to override.

### PR DESCRIPTION
**Pull request by ilayaperumalg**
Resolves #1658

### CODE REVIEW

File: spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -137,6 +137,9 @@ Like in the above example, the remote repositories can be specified along with t
 The repository policies can be specified per remote repository configuration as above.
 The key `policy` is applicable to both `snapshot` and the `release` repository policies.
 
+You can refer https://github.com/eclipse/aether-core/blob/4cf5f7a406b516a45d8bf15e7dfe3fb3849cb87b/aether-api/src/main/java/org/eclipse/aether/repository/RepositoryPolicy.java#L16[Repository Policies] for the list of
+available repository policies.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc]: "supported" instead of "available"?

Comment (sabbyanandan) [reply spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc]: Looks good, o'wise.

**Comment by cppwfs** [comment]
LGTM.  Will add @sabbyanandan comments on merge.

**Comment by cppwfs** [comment]
Applied changes, Rebased, Squashed, Merged.

**Pull request closed without merging**